docs/404.html

```diff
@@ -32,6 +32,19 @@
 <meta name="twitter:description" content="Documentation for Kubectl and Kustomize"/>
 
 
+<script type="application/javascript">
```

Comment:
Why the hugo adds this GA codes.

Reply:
strange, i don't see evidence of local changes since your helm pr.

perhaps hugo is pulling in some remote assets?  this do-not-track stuff looks generic